data_structures/src/mainnet_validations.rs

```diff
@@ -209,6 +209,32 @@ pub fn after_second_hard_fork(epoch: Epoch, environment: Environment) -> bool {
     epoch >= SECOND_HARD_FORK && Environment::Mainnet == environment
 }
 
+/// Allows to check the active Witnet Improvement Proposals
+#[derive(Clone, Debug)]
+pub struct ActiveWips {
+    pub active_wips: HashMap<String, Epoch>,
+    pub block_epoch: Epoch,
+    pub environment: Environment,
+}
+
+impl ActiveWips {
+    pub fn first_hard_fork(&self) -> bool {
+        after_first_hard_fork(self.block_epoch, self.environment)
+    }
+
+    pub fn second_hard_fork(&self) -> bool {
```

Comment:
```suggestion
   // WIPs 0009, 0011 and 0012 were activated through community coordination on April 28, 2021
    pub fn wips_0009_0011_0012(&self) -> bool {
```

Reply:
It would be nice to create 3 separate functions, one for each WIP

Reply:
Why so if they were activated atomically?

Reply:
Just so that when we use it in the code, we explicitly say which WIP introduced for that piece of code?

Reply:
Yes, exactly, for documentation purposes. But we can do that later because this PR already changes many lines.